docs/bom/create-a-bill-of-materials.md

```diff
@@ -0,0 +1,251 @@
+manifest that lists everything included in a software release.
+"Everything" can take different meanings: software packages or images,
+documentation, tarballs, single files. These pieces can be components
+or source code, variants of the same artifact (eg a binary for different
+platforms).
+
+An SBOM can also provide visibility on the dependencies of your project.
+There are many types of dependencies and many reasons that consumers of
+a project need to know them: security,  compliance, compatibility.
+
+Finally, a Bill of Materials allows software creators to express licensing
+information for their project as a whole, but also for individual pieces
+and it's dependencies. You can release your project under the Apache 2.0
+license but have its documentation published under Creative Commons. Then,
+there are all of your dependencies' original licenses. A well-written
+SBOM can express all of them in the same document. 
+
+### bom: A tool to generate SBOMs for your Project
+
+As part of the effort to produce a bill of materials for Kubernetes, SIG
+Release developed a set of libraries to produce fully compliant SPDX SBOMs.
+Our tools support license scanning, image layer analyzers, processing of
+golang dependencies, and other features. These libraries are available for
+other projects to automate the production of their own Bills of Materials.
+
+For simpler use cases, all of our SBOM automation is also available in
+a general-purpose tool called `bom`. You can find all the options that
+`bom` supports in its README.md.
+
+`bom` supports generating Bills of Materials in SPDX compliant tag-value
+format. It can process directories, single files, read the contents of
+container images (both from docker tar archives and directly from registries),
+and tarred sources.
+
+In addition `bom` will scan your code to find licensing information. Its
+classifier supports detection of all the SPDX recognized licenses.
+
+## SPDX: Software Package Data Exchange
+
+SPDX or _Software Package Data Exchange_ is an open standard to create
+bills of materials. It has been in the works for 10+ years, coordinated
+by the SPDX Workgroup, a project of the Linux Foundation. 
+
+As of June 2020, the SPDX spec is in version 2.2. The current version
+allows software authors to include metadata about their project describing
+its contents, relationships among them, and other components and licensing. 
+
+### Files and Packages
+
+There are two main building blocks in an SPDX manifest: Files and Packages.
+
+Files are what you would expect: an individual item in a filesystem tree.
+The data about a file in an SPDX SBOM includes its name, checksums, license,
+file type, copyright data, and other attributes.
+
+Packages are a non-specific element in SPDX representing anything that can
+group other elements. An `.rpm` or `.deb` package can be an SPDX package, but
+so can be a docker image or a tarball. Packages contain files, but can also
```

Comment:
```suggestion
so can be a container image or a tarball. Packages contain files, but can also
```
I'll not highlight more of those since it will occur in multiple places in the following paragraphs. I assume it works with OCI images in the same way as Docker v2 manifests, right? Then we should always refer the OCI format aka the term "container image".

Reply:
Absolutely right. I have to change that automatic behavior of always writing "docker image". I changed all of them in the dock.